packages/cli/src/utils/config.ts

```diff
@@ -22,7 +22,13 @@ export function configPath(configDir: string) {
 
 export function readConfig(configDir: string): CeloConfig {
   if (fs.pathExistsSync(configPath(configDir))) {
-    return fs.readJSONSync(configPath(configDir))
+    const existingConfig = fs.readJSONSync(configPath(configDir))
+    const combinedConfig = { ...existingConfig, ...defaultConfig }
+    if (combinedConfig.hasOwnProperty('nodeUrl')) {
+      /* tslint:disable:no-string-literal */
+      combinedConfig['node'] = combinedConfig['nodeUrl']
```

Comment:
would this work? `combinedConfig.node = combinedConfig.nodeUrl`

I think then no tslint disabled is needed.

Reply:
Adding a little to what Aaron said,
Typescript when you've checked that an object already has a property, in that block of the if, it will assume that the property is in the object. So, it will let you do `combinedConfig.nodeUrl` without any warning.

So yes, in this case, you could use it like this: `combinedConfig.node = combinedConfig.nodeUrl`

Reply:
great catch, fixed!